CONTRIBUTING.md

```diff
@@ -250,29 +250,34 @@ and follow [those instructions](https://github.com/elastic/docs#for-a-local-repo
 
 If you have access to make releases, the process is as follows:
 
+For illustration purpose, `1.2.3` will be the target release version, and the git remote will be `upstream`.
+
 1. Check if sonatype is up: https://status.maven.org
-1. Review project version. The release version will be `${project.version}` without the `-SNAPSHOT`.
-   1. In case you want to update the version, execute `mvn versions:set -DnewVersion=${VERSION}` where `${VERSION}` is the target version.
+1. Update [`CHANGELOG.asciidoc`](CHANGELOG.asciidoc) to reflect the new version release:
+   1. Go over PRs or git log and add bug fixes and features.
+   1. Move release notes from the `Unreleased` sub-heading to the correct `[[release-notes-{major}.x]]` sub-heading ([Example PR](https://github.com/elastic/apm-agent-java/pull/1027/files) for 1.13.0 release).
+1. Review Maven project version, you must have `${project.version}` equal to `1.2.3-SNAPSHOT`, `-SNAPSHOT` suffix will be removed during release process.
+   1. If needed, use following command to set version `mvn versions:set -DnewVersion=1.2.3`, then commit and push changes.
```

Comment:
Shouldn't it be ``mvn versions:set -DnewVersion=1.2.3-SNAPSHOT`?

Reply:
good catch, will fix this directly on master then.